sunbird-rc-esignet-integration-impl/pom.xml

```diff
@@ -92,6 +92,20 @@
 			<version>1.7.30</version>
 			<scope>provided</scope>
 		</dependency>
+		<!-- https://mvnrepository.com/artifact/org.apache.velocity/velocity -->
+		<dependency>
+			<groupId>org.apache.velocity</groupId>
+			<artifactId>velocity</artifactId>
+			<version>1.7</version>
+		</dependency>
+
+		<dependency>
+			<groupId>org.json</groupId>
+			<artifactId>json</artifactId>
+			<version>20210307</version>
+		</dependency>
```

Comment:
Do we really need this new json dependency, can't we use the existing json libraries ?

Reply:
i will check later

Reply:
I don't remember seeing any code using this dependency, if you want to check this later, please create a jira sub task and provide the link.

Reply:
i will use this in VCI plugin

Reply:
removed